dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -166,14 +166,14 @@ decompression.tolerance.reached=CWWKT0034E: The HTTP channel attempted to decomp
 
-cookies.samesite.duplicateName=CWWKT0035W: A duplicate cookie name or pattern [{0}] was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored for all configuration lists. Any cookie name or pattern that is defined by the ��lax��, ��none��, and ��strict�' configurations can be defined in only one of the three configurations.
+cookies.samesite.duplicateName=CWWKT0035W: A duplicate cookie name or pattern [{0}] was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored for all configuration lists. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations.
 cookies.samesite.duplicateName.explanation=The specified value was already found by the configuration and mapped to a SameSite value.
-cookies.samesite.duplicateName.useraction=Ensure that cookie names and patterns are unique across the 'lax', 'none', and 'strict' configuration lists.
+cookies.samesite.duplicateName.useraction=Ensure that cookie names and patterns are unique across the lax, none, and strict configuration lists.
 
 cookies.samesite.unsupportedWildcard=CWWKT0036W: An unsupported use of the wildcard character was attempted by the value [{0}]. The SameSite configuration is not set for this value.
 cookies.samesite.unsupportedWildcard.explanation=The wildcard (*) character was used in an unsupported way. Support consists of a single wildcard (*) character that is used as a stand-alone value, or following any number of cookie name prefixes.
-cookies.samesite.unsupportedWildcard.useraction=Ensure that only a single wildcard (*) character is used per pattern in the 'lax', 'none', and 'strict' lists. Additionally, use of this character is supported only under the following two conditions: as a stand-alone value or following cookie name prefixes.
+cookies.samesite.unsupportedWildcard.useraction=Ensure that only a single wildcard (*) character is used per pattern in the lax, none, and strict lists. Additionally, use of this character is supported only under the following two conditions: as a stand-alone value or following cookie name prefixes.
 
-cookies.samesite.knownDuplicateName=CWWKT0037W: A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the ��lax��, ��none��, and ��strict�' configurations can be defined in only one of the three configurations.�
+cookies.samesite.knownDuplicateName=CWWKT0037W: A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations.
```

Comment:
Change,
"A cookie name or pattern [{0}], which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations."
to,
"A  [{0}] cookie name or pattern, which is marked as a duplicate, was found in the SameSite [{1}] configuration. The [{0}] cookie name or pattern is ignored. Any cookie name or pattern that is defined by the lax, none, and strict configurations can be defined in only one of the three configurations."

Reply:
Discussed internally through Slack, marking this as resolved.